instrumentation/gruf/lib/opentelemetry/instrumentation/gruf/interceptors/client.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+module OpenTelemetry
+  module Instrumentation
+    module Gruf
+      module Interceptors
+        class Client < ::Gruf::Interceptors::ClientInterceptor
+          # rubocop:disable Metrics/MethodLength
+          def call(request_context:)
+            return yield if instrumentation_config.blank?
```

Comment:
For example, I enable opentelemetry only for a certain environment, and in the case of interceptors, middlewares, I also need to connect an interceptor/middleware for a certain environment, I don’t want this, which means that if the config is empty, then open telemetry is not enabled

Reply:
I will admit that I still don't understand precisely how we could get into such a state; but I will also admit that I don't know the Gruf framework very well and so that might just be my ignorance showing. 😆

Reply:
Does this instrumentation depend on ActiveSupport or did `blank?` get added to Ruby?

Reply:
this instrumentation does not depend on ActiveSupport.

'I will admit that I still don't understand precisely how we could get into such a state; but I will also admit that I don't know the Gruf framework very well and so that might just be my ignorance showing.' - sometimes we turn off temporarily necessary instrumentation, example, :

`
OpenTelemetry::SDK.configure do |c|
  if config.instrumentaion_gruf_enabled?
      c.use 'OpenTelemetry::Instrumentation::Gruf', {
        peer_service: "Example",
        grpc_ignore_methods_on_client: [],
        grpc_ignore_methods_on_server: [],
        allowed_metadata_headers: [],
      }
  end
  `
  
  and we wouldn't want to edit the code like:
`
  client_options = { interceptors: [OpenTelemetry::Instrumentation::Gruf::Interceptors::Client.new] }
  client = Gruf::Client.new(
     service: Proto::Example::ExampleAPI, options: {}, client_options: client_options
  )
`